spec/controllers/v1.0/requests_controller_spec.rb

```diff
@@ -459,7 +459,7 @@
       [{
         'app_name'    => 'app1',
         'object_type' => 'otype1',
-        'tags'        => [{'namespace' => 'ns1', 'name' => 'name1', 'value' => 'v1'}]
+        'tags'        => [{:tag => '/ns1/name1=v1'}]
```

Comment:
prefer `'tag'` other than `:tag`

Reply:
I would like to define request body to have `"tags": ["/ns1/name1=v1", "/ns2/name2=v2"]`
I understand the app that prepares the request body get tags from other API with the format `[{"tag": "/ns1/name1=v1"}, {"tag":"/ns2/name2=v2"}]`. Approval-api has to accept the redundant format if it does not want the caller to have the burden to do a format transformation.

Reply:
@bzwei @hsong-rh 
The multiple tag is  from the Tagging changes implemented here
https://github.com/RedHatInsights/topological_inventory-api/blob/784bc2640efd190d3c43b3ce5ebaa16f6125906a/public/doc/openapi-3-v2.0.0.json#L10696